<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLMBaseDockHeader</class>
 <widget class="QWidget" name="PLMBaseDockHeader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">  QComboBox {
      border: 2px solid transparent;
      min-width: 6em;
      max-width: 20em;
	
	background-color: rgba(255, 255, 255, 0);
	color: black;

  }
  QComboBox::drop-down {
    border: 0px solid transparent;
    subcontrol-origin: padding;
      subcontrol-position: top right;

}
  QComboBox:on { /* shift the text when the popup opens */
      padding-top: 3px;
      padding-left: 4px;
  }

  QComboBox::drop-down:!editable {
background: transparent;
	color: black;
  }

  QComboBox:!editable, QComboBox::drop-down:editable {
background: transparent;
	color: black;
  }

/*   QComboBox gets the &quot;on&quot; state when the popup is open */
  QComboBox:!editable:on, QComboBox::drop-down:editable:on {
background: grey;
	color: black;
  }
  QComboBox::down-arrow {
      image: url(:/pics/go-down.svg);
      width: 20px;
      height: 20px;

  }
  QComboBox::down-arrow:on { /* shift the arrow when popup is open */
      top: 1px;
      left: 1px;
  }
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="dockHeaderComboBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="headerHolder">
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QToolButton" name="addDockButton">
     <property name="text">
      <string>+</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="closeButton">
     <property name="text">
      <string>x</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
